tsconfig.json

```diff
@@ -4,11 +4,13 @@
     "esModuleInterop": true,
     "module": "commonjs",
     "moduleResolution": "node",
-    "noEmit": true,
+    "outDir": "lib",
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
     "target": "es2020",
     "typeRoots": ["node_modules/@types", "typings"]
-  }
+  },
+  "include": ["src"],
+  "exclude": ["node_modules", "**/__tests__/*"]
```

Comment:
Is the exclude needed if we only include `src`?

Reply:
You are right. Probably not.

Reply:
I don't think either `include` or `exclude` are needed. See my comment here: https://github.com/gnosis/gp-v2-contracts/pull/216#issuecomment-738843828